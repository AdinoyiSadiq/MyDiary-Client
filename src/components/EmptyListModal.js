import React from 'react';
import { Link } from 'react-router-dom';

const EmptyListModal = () => (
  <section className="authContent emptyList">
    <div className="title">You don't seem to have created an entry</div>
    <hr/>
    <form>
      <Link to="/main/addEntry">
        <button>Create an Entry</button>
      </Link>
    </form>
	</section>
);

export default EmptyListModal;