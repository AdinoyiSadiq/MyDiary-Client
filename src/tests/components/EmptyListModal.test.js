import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EmptyListModal from '../../components/EmptyListModal';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<EmptyListModal />);
});


describe('EmptyListModal UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});