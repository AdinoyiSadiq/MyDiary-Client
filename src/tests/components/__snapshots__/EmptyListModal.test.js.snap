// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyListModal UI render features view should render as expected 1`] = `
<section
  className="authContent emptyList"
>
  <div
    className="title"
  >
    You don't seem to have created an entry
  </div>
  <hr />
  <form>
    <Link
      replace={false}
      to="/main/addEntry"
    >
      <button>
        Create an Entry
      </button>
    </Link>
  </form>
</section>
`;
